internal/component/mimir/rules/kubernetes/events_test.go

```diff
@@ -251,6 +251,18 @@ func TestAdditionalLabels(t *testing.T) {
 		metrics:           newMetrics(),
 		logger:            log.With(log.NewLogfmtLogger(os.Stdout), "ts", log.DefaultTimestampUTC),
 		externalLabels:    map[string]string{"foo": "bar"},
+		extraQueryLabels: []queryLabel{
+			{
+				name:      "cluster",
+				matchType: "=",
+				value:     "big",
+			},
+			{
+				name:      "job",
+				matchType: "=",
+				value:     "good",
+			},
+		},
```

Comment:
Users would be expected to set something like this in their alloy config for mimir.rules.kubernetes.